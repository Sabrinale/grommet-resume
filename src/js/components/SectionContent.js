import React from 'react'

import { Box } from 'grommet'

export const SectionContent = ({ children }) => (
  <Box
    align="start"
    pad={{ left: '0px', right: '0px', top: '0px', bottom: '0px' }}
    margin={{ left: '0px', right: '0px', top: '0px', bottom: 'medium' }}
  >
    <Box
      fill={true}
      align="start"
      overflow="hidden"
      border={{ side: 'left', color: 'brand' }}
      elevation="none"
      round="none"
      gap="small"
      pad={{ left: 'medium' }}
      margin={{}}
    >
      <Box fill={true} flex={true}>
        {children}
      </Box>
    </Box>
  </Box>
)
