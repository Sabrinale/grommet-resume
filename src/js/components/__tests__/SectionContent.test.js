import React from 'react'
import renderer from 'react-test-renderer'

import { SectionContent } from '../SectionContent'

test('renders section content element', () => {
  const component = renderer.create(<SectionContent>test</SectionContent>)
  let tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
