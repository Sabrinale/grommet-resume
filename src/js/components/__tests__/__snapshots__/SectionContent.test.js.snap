// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders section content element 1`] = `
<div
  className="StyledBox-sc-13pk1d4-0 jfnaIv"
>
  <div
    className="StyledBox-sc-13pk1d4-0 dViohz"
  >
    <div
      className="StyledBox-sc-13pk1d4-0 OaRuD"
    >
      test
    </div>
  </div>
</div>
`;
